src/main/java/org/broadinstitute/hellbender/exceptions/UserException.java

```diff
@@ -112,18 +110,6 @@ public CannotHandleGzippedRef() {
     public static class CouldNotCreateOutputFile extends UserException {
         private static final long serialVersionUID = 0L;
 
-        public CouldNotCreateOutputFile(File file, String message, Exception e) {
-            super(String.format("Couldn't write file %s because %s with exception %s", file.getAbsolutePath(), message, getMessage(e)));
-        }
-
-        public CouldNotCreateOutputFile(File file, String message) {
-            super(String.format("Couldn't write file %s because %s", file.getAbsolutePath(), message));
-        }
-
-        public CouldNotCreateOutputFile(String filename, String message, Exception e) {
-            super(String.format("Couldn't write file %s because %s with exception %s", filename, message, getMessage(e)));
-        }
```

Comment:
These overloads will be needed as we port tools -- there's no good reason to delete them.